packages/db/src/types/file.ts

```diff
@@ -1,19 +1,22 @@
 export interface PostgresFile {
   bucket?: string,
   key?: string,
-  granule_cumulus_id: number
-  checksum_type?: string
-  checksum_value?: string
-  file_name?: string
-  file_size?: number
-  path?: string
-  source?: string
+  granule_cumulus_id: number,
+  checksum_type?: string,
+  checksum_value?: string,
+  file_name?: string,
+  file_size?: number,
+  path?: string,
+  source?: string,
 }
 
-export interface PostgresFileRecord extends PostgresFile {
+// file_size is stored as a BigInt in Postgres. It returns from PG to Node
+// as a "string" type.
```

Comment:
https://wiki.earthdata.nasa.gov/pages/viewpage.action?pageId=217416307

Reply:
:thumbsup: good use of Omit.